Ginger/GingerCore/Resources/GingerHTMLHelper.js

```diff
@@ -219,10 +213,11 @@ function define_GingerLib() {
 
 			if(img!=undefined)
 			{
-				if (img.indexOf("ERROR") == -1)
-					pl = new GingerPayLoad("HTMLScreenShot");
-				else
-					pl = new GingerPayLoad("Error");
+                if (img.indexOf("ERROR") == -1)
+                    pl = new GingerPayLoad("HTMLScreenShot");
+                else {
+                    pl = new GingerErrorPayLoad(0, "Error");
```

Comment:
This will fail In case of Error payload we are already closing a payload inside GingerErrorPayload function